packages/x-components/tests/e2e/cucumber/url/url.spec.ts

```diff
@@ -21,3 +24,17 @@ When('navigating back', () => {
 When('clicking result in position {int}', (index: number) => {
   cy.getByDataTest('result-link').eq(index).click();
 });
+
+// Scenario 4
+Then("url doesn't contain parameter {string} with value {string}", (key: string, value: string) => {
```

Comment:
Shouldn't this be an `And` instead of a `Then`?

```suggestion
And("url doesn't contain parameter {string} with value {string}", (key: string, value: string) => {
```